server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/SetMessagesCreationProcessor.java

```diff
@@ -106,32 +109,78 @@ public SetMessagesResponse process(SetMessagesRequest request, MailboxSession ma
         Mailbox<Id> outbox;
         try {
             outbox = getOutbox(mailboxSession).orElseThrow(() -> new MailboxRoleNotFoundException(Role.OUTBOX));
-        } catch (MailboxException | MailboxRoleNotFoundException e) {
+        } catch (MailboxRoleNotFoundException e) {
             LOGGER.error("Unable to find a mailbox with role 'outbox'!");
             throw Throwables.propagate(e);
         }
 
         // handle errors
```

Comment:
looks like a good candidate for another method (comments in a too long method is a good sign)